crates/ruff_linter/src/rules/pyflakes/snapshots/ruff_linter__rules__pyflakes__tests__preview__F401_F401_25__all_nonempty____init__.py.snap

```diff
@@ -0,0 +1,46 @@
+---
+source: crates/ruff_linter/src/rules/pyflakes/mod.rs
+---
+__init__.py:19:8: F401 [*] `sys` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
+   |
+19 | import sys  # F401: remove unused
+   |        ^^^ F401
+   |
+   = help: Remove unused import: `sys`
+
+ℹ Unsafe fix
+16 16 | import argparse  # Ok: is exported in __all__
+17 17 | 
+18 18 | 
+19    |-import sys  # F401: remove unused
+20 19 | 
+21 20 | 
+22 21 | # first-party
+
+__init__.py:36:15: F401 [*] `.unused` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
+   |
+36 | from . import unused # F401: add to __all__
+   |               ^^^^^^ F401
+   |
+   = help: Add unused import to __all__: `.unused`
```

Comment:
Hmm, not sure about this one. The help message sounds like it's literally going to add the string `".unused"` to `__all__` -- but it's actually going to add the string `"unused"` to `__all__`. I think one of the following two options would be much better:

```suggestion
__init__.py:36:15: F401 [*] `.unused` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
   |
36 | from . import unused # F401: add to __all__
   |               ^^^^^^ F401
   |
   = help: Add `unused` to `__all__`
```

or 

```suggestion
__init__.py:36:15: F401 [*] `.unused` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
   |
36 | from . import unused # F401: add to __all__
   |               ^^^^^^ F401
   |
   = help: Add `"unused"` to `__all__`
```

Reply:
SGTM